engine/cache/manager.go

```diff
@@ -388,8 +388,8 @@ func (m *manager) Import(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	importedCache := solver.NewCacheManager(ctx, m.ID(), keyStore, resultStore)
-	newInner := solver.NewCombinedCacheManager([]solver.CacheManager{importedCache}, m.localCache)
+	importedCache := solver.NewCacheManager(ctx, m.ID()+"-import", keyStore, resultStore)
```

Comment:
changing the name here isn't strictly necessary, just helps with the logging I added to buildkit while debugging this